File: Xiaomi6/app/src/main/java/com/example/xiaomi6/MyPageFragment.java
package com.example.xiaomi6;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

public class MyPageFragment extends Fragment {
    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        //         Inflate the layout for this fragment
        View view = inflater.inflate(R.layout.my_page, container, false);
        return view;
    }
}

File: Xiaomi11/app/src/main/java/com/example/xiaomi11/fragment/MyPageFragment.kt
package com.example.xiaomi11.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.example.xiaomi11.R

class MyPageFragment : Fragment() {
    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        //         Inflate the layout for this fragment
        return inflater.inflate(R.layout.my_page, container, false)
    }
}